packages/xod-arduino/platform/nodes/record.tpl.cpp

```diff
@@ -0,0 +1,27 @@
+node {
+    meta {
+        struct Type {
+            {{#each inputs}}
+            {{#if (isConstantType type)}}static constexpr {{/if~}}
+            typeof_{{ pinKey }} {{ pinKey }}
```

Comment:
A common prefix is required.

```suggestion
            typeof_{{ pinKey }} field_{{ pinKey }}
```

Otherwise, a xoder can easily break his C++ program by naming a pin as a keyword (e.g., `void`) o macro (`M_PI`)